nsqd/protocol_v2.go

```diff
@@ -92,6 +93,7 @@ func (p *protocolV2) IOLoop(conn net.Conn) error {
 
 			sendErr := p.Send(client, frameTypeError, []byte(err.Error()))
 			if sendErr != nil {
+				p.ctx.nsqd.logf("ERROR: [%s] - %s%s", client, sendErr, ctx)
```

Comment:
Probably not. `Exec` is going to return a `FatalClientErr` and we don't want to overwrite it on our way out.